resources/js/Components/icons/Calendar.svelte

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="lg:w-7 lg:h-7 w-6 h-6" >
```

Comment:
Remember to pass in the svelte global props.class into the class here, so we can overwrite the classes if we want.